packages/back-end/src/util/secrets.ts

```diff
@@ -11,6 +11,9 @@ if (fs.existsSync(".env.local")) {
 
 export const IS_CLOUD = !!process.env.IS_CLOUD;
 
+import { verifyCloud } from "enterprise";
```

Comment:
nitpick - is there any reason for not placing this `import` statement at the top of the file? stylistically it seems anomalous